CONTRIBUTING.md

```diff
@@ -136,16 +134,15 @@ Adding a new listing to the Kong Hub may be proposed by:
 
 1. Clone this repo
 1. Create a publisher directory at`_app/_hub/`, such as
- `_app/_hub/your-github-handle` (if you are contributing as an individual)
- or `_app/_hub/company-name` (if you are contributing as a company). See
- other Kong Hub listings for examples of publisher names.
+ `_app/_hub/your-GitHub-handle` (if you are contributing as an individual)
+ or `_app/_hub/company-name` (if you are contributing as a company). See other Kong Hub listings for examples of publisher names.
```

Comment:
What is the purpose of this change?

Reply:
Same as above, it was shifting to the new line.